plugins/link.md

```diff
@@ -209,9 +252,18 @@ tinymce.init({
 
 ### `target_list`
 
-The `target_list` lets you specify a predefined list of targets for the `link` dialog. This defaults to a dialog containing the options `_self` and `_blank`.
+The `target_list` option lets you specify a list of named targets for the `link` dialog. These targets will appear in a dropdown menu in the link dialog. Each target must be defined as an object with a `title` and a `value`. For example: `{title: 'Same page', value: '_self'}`. When the dialog is submitted, the `value` of the selected target item will be set as the link's `target` attribute.
 
-**Type:** `String`
+If [`default_link_target`](#default_link_target) is also configured and its value matches a value specified by `target_list`, that item will be set as the default item for the targets dropdown in the link dialog.
+
+**Default:**
+
+```js
+[
+  { text: 'Current window', value: '' },
+  { text: 'New window', value: '_blank' }
+]
+```
 
 **Example, adding a `_lightbox` target to the dropdown list**
```

Comment:
make this comment valid. the   {title: 'Lightbox', value: '_lightbox'} config below was deleted

Reply:
updated link.md with ce1b39d7dfb99305583d130f081ea6cc597a3d50 to remove references to lightbox which is not a valid target